lib/resque/scheduler/failure.rb

```diff
@@ -0,0 +1,16 @@
+module Resque
+  module Scheduler
+    module Failure
+      # allow user to set an additional failure handler
+      attr_writer :failure_handler
```

Comment:
I think this would be more discoverable if it just lived in Resque::Scheduler, which is the only class that includes this module.  Moving code into a module when it is only included into a single class is an anti-pattern.